File: exocps.kt
import java.io.File
import kotlin.system.exitProcess

const val FILE_PATH = "payments.txt"

fun printMenu() {
    println(
        """
        *********************************
        *  Online College Payment System *
        *********************************
        1. Process Payment
        2. Delete Payment
        3. Edit Payment
        4. Search Payment
        5. Print All Payments
        6. Exit
    """.trimIndent()
    )
}

fun makePayment() {
    print("Enter Student ID: ")
    val id = readLine()!!
    print("Enter Amount: RM")
    val amount = readLine()!!
    
    File(FILE_PATH).appendText("$id,$amount\n")
    println("Payment saved!")
}

fun deletePayment() {
    print("Enter Student ID to delete: ")
    val targetId = readLine()!!
    
    val lines = File(FILE_PATH).readLines()
        .filterNot { it.startsWith("$targetId,") }
    
    File(FILE_PATH).writeText(lines.joinToString("\n"))
    println("Payment deleted!")
}

fun main() {
    File(FILE_PATH).apply { if (!exists()) createNewFile() }
    
    while (true) {
        printMenu()
        when (readLine()) {
            "1" -> makePayment()
            "2" -> deletePayment()
            "6" -> exitProcess(0)
            else -> println("Invalid choice!")
        }
    }
}

File: exocps.js
const fs = require('fs');
const readline = require('readline');

// File operations
const FILE_PATH = 'payments.txt';

// Create file if missing
if (!fs.existsSync(FILE_PATH)) {
    fs.writeFileSync(FILE_PATH, '');
}

// Main menu
function displayMenu() {
    console.log(`
*****************************************************************
*             Online College Payment System                     *
*****************************************************************
*                                                               *
*        1 - Process a Payment                                  *
*        2 - Delete Payment                                     *
*        3 - Edit Payment                                       *
*        4 - Search Payment                                     *
*        5 - Print Payments                                     *
*        6 - Exit                                               *
*                                                               *
*****************************************************************`);
}

// Input helper
async function askQuestion(question) {
    const rl = readline.createInterface({
        input: process.stdin,
        output: process.stdout
    });
    return new Promise(resolve => {
        rl.question(question, answer => {
            rl.close();
            resolve(answer);
        });
    });
}

// CRUD operations
async function makePayment() {
    const studentId = await askQuestion('Enter Student ID: ');
    const amount = await askQuestion('Enter Payment Amount: RM');
    fs.appendFileSync(FILE_PATH, `${studentId},${amount}\n`);
    console.log('\nPayment processed successfully!\n');
}

function deletePayment(studentId) {
    const payments = fs.readFileSync(FILE_PATH, 'utf-8').split('\n');
    const newPayments = payments.filter(line => {
        const [id] = line.split(',');
        return id !== studentId;
    });
    fs.writeFileSync(FILE_PATH, newPayments.join('\n'));
    console.log('Payment deleted!');
}

// (Other functions: edit, search, print similar to above)

// Main loop
async function main() {
    while (true) {
        displayMenu();
        const choice = await askQuestion('Enter your choice: ');

        switch (choice) {
            case '1': await makePayment(); break;
            case '2': {
                const id = await askQuestion('Enter Student ID to delete: ');
                deletePayment(id);
                break;
            }
            // Add cases 3-6 similarly
            case '6': process.exit(0);
            default: console.log('Invalid choice!');
        }
    }
}

main();